feat(ci): move to mc-modpack-kit

jh-devv opened the pull request.

jh-devv commented:
@Aninuscsalas You have to set the Modrinth ID and TOKEN in the settings
Also allow "Allow PR's to be managed with actions"

jh-devv commented:
@Aninuscsalas Can't maybe yet merge cause https://github.com/jh-devv/mc-modpack-kit/issues/49

jh-devv commented:
@Aninuscsalas This should be pretty much complete (still have to add some stuff) but I want to ask something:

Does the version of the pack need to be different for the Forge and Quilt version?
Because for that, we would need https://github.com/jh-devv/mc-modpack-kit/issues/49

Aninuscsalas commented:
> Does the version of the pack need to be different for the Forge and Quilt version? Because for that, we would need [jh-devv/mc-modpack-kit#49](https://github.com/jh-devv/mc-modpack-kit/issues/49)

I'm still not sure what to do. I think its better to wait because in my test repos modrinth publishing/builing often fails, or skips everything. Im going to some testing with other packs before this one gets merged

jh-devv commented:
Hmm, weird? Have you read the README?

jh-devv commented:
> > Does the version of the pack need to be different for the Forge and Quilt version? Because for that, we would need [jh-devv/mc-modpack-kit#49](https://github.com/jh-devv/mc-modpack-kit/issues/49)
> 
> I'm still not sure what to do. I think its better to wait because in my test repos modrinth publishing/builing often fails, or skips everything. Im going to some testing with other packs before this one gets merged

Could you share a test repo of yours?

jh-devv commented:
This theorically should be now working (the release please things just means you can bump one part of the monorepo)
Shoudl work!

Aninuscsalas commented:
> This theorically should be now working (the release please things just means you can bump one part of the monorepo) Shoudl work!

Should we try merging it? I have added the secrets to the repo. I guess the versioning can restart from 0.1.0... Im still not sure how should i style commit names, chore(quilt):,  and feat(forge): can work fine but im not sure that listing all the updated mods separately or just saying "Updated mods" is a better idea.

jh-devv commented:
> Should we try merging it? I have added the secrets to the repo. I guess the versioning can restart from 0.1.0... Im still not sure how should i style commit names, chore(quilt):, and feat(forge): can work fine but im not sure that listing all the updated mods separately or just saying "Updated mods" is a better idea.

@Aninuscsalas Yea, but you could just try to run the version workflow right away and set it to just some value (e.g. fabric, it does not support currentry different versions for different components) and for the version set `0.1.0` if you want that!

And for the mod commits, I would use it like:
For updates:
`chore(fabric): Update mods`
For bigger mod changes (e.g. adding or removing)
`feat(fabric): Added mods` or `feat(fabric): Added Fabric API`
Its pretty much how'd you like it!

Hope this helps
*jh-devv*

Aninuscsalas commented on quilt/mods/vow.pw.toml:
This was a bad idea, this needs an access code

jh-devv replied:
Yea.. Modrinth does not allow download from "custom" urls

Aninuscsalas replied:
nono, its on modrinth now https://modrinth.com/mod/vow, it just needs an acces key to play it

jh-devv replied:
Oh, cool :3

jh-devv replied:
> nono, its on modrinth now https://modrinth.com/mod/vow, it just needs an acces key to use it

Is it still up?

jh-devv replied:
![Screenshot_20230926_143834](https://github.com/Aninuscsalas/wynncraft-101/assets/122896463/d73a94be-c6e3-4428-af17-b4d7556f801b)
I asked Rafii about this, it was made Public by accident and thats why I didn't work

jh-devv commented:
@Aninuscsalas Now my project uses commits like
- `feat(modpack/<mod>): <description>` for adding new features, configs or content.
- `chore(modpack/<mod>): <description>` for routine maintenance tasks or cleanups.
- `update(modpack/<mod>): update <mod>` for updating mods to newer versions.
- `mod(modpack/<mod>): add <mod>` for introducing new mods to the modpack.

Aninuscsalas commented:
I'm gonna probably try to merge this this in the next few days 🙃

jh-devv commented:
@Aninuscsalas now just run the set version workflow ;3

Aninuscsalas commented:
> @Aninuscsalas now just run the set version workflow ;3

can i do that before merging? (im new to github)

jh-devv commented:
> > @Aninuscsalas now just run the set version workflow ;3
> 
> can i do that before merging? (im new to github)

You can do right after or I can do it :3

jh-devv commented:
https://github.com/orgs/community/discussions/25488 @Aninuscsalas
